tsl/test/expected/jit-13.out

```diff
@@ -0,0 +1,225 @@
+-- This file and its contents are licensed under the Timescale License.
```

Comment:
Not able to get github to display this file correctly. Summarizing the changes here:
 at line 2317:  we have plans without indexcond. I think this is similar to the other diffs seen int transparent_dcompression_ordered_index.out